test/integration/guest_user_test.rb

```diff
@@ -20,9 +21,9 @@ def setup
                             email: 'jwan622@example.com',
                             role: 0)
 
-    @item1 = Item.create(title: 'espresso', price: 9000)
+    @item1 = Item.create(title: 'espresso', description: "this is black gold", price: 30000)
```

Comment:
Line is too long. [92/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.